chia/plotting/manager.py

```diff
@@ -431,7 +431,7 @@ def plot_removed(test_path: Path):
                     del self.plot_filename_paths[filename]
 
             initial_value: Dict[Path, PlotInfo] = {}
-            self.plots = reduce(reduce_function, executor.map(process_file, all_filenames), initial_value)
+            self.plots = reduce(reduce_function, executor.map(process_file, plot_paths), initial_value)
```

Comment:
`all_filenames` was renamed to `plot_paths`, but they still contain the same data, right?

Reply:
Correct